index.html

```diff
@@ -5155,6 +5155,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
 
 </section>
 
+<section id="presentation-media-type">
+  <h2>The <code>application/presentation+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/presentation+ld+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Presentations format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>presentation+ld+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/presentation+ld+json</code>"
+        Media Type are required to conform to all of the requirements
+        for the "<code>application/ld+json</code>" Media Type and are
+        therefore subject to the same encoding considerations specified
+        in Section 11 of [[!RFC7159]].
+      </td>
+    </tr>
+    <tr>
+      <td>Security considerations: </td>
+      <td>As defined in this specification.</td>
+    </tr>
+    <tr>
+      <td>Contact: </td>
+      <td>
+        W3C Verifiable Credentials Working Group <a href="mailto:public-vc-wg@w3.org">public-vc-wg@w3.org</a>
+      </td>
+    </tr>
+  </table>
+
+  <p>
+    Note that while the Verifiable Presentations format uses JSON-LD
+    conventions, there are a number of constraints and additional
+    requirements for Verifiable Presentation implementations that justify
```

Comment:
It seems that @Sakurann's [question above](https://github.com/w3c/vc-data-model/pull/1032/files#r1103875272) applies here as well.  I'm also noticing an unexplained flipping between Presentation singular and Presentations plural, for which it seems we should pick one.
```suggestion
    Note that while the Presentations format uses JSON-LD
    conventions, there are a number of constraints and additional
    requirements for Presentation implementations that justify
```

Reply:
I'm holding on merging this, until folks way in on https://github.com/w3c/vc-data-model/pull/1032#discussion_r1106605757

Reply:
I've raised issue #1052 to track @Sakurann and @TallTed's concerns... which should allow us to merge this PR as long as they are ok w/ that approach.

Reply:
I agree on using the singular "Presentation" rather than "Presentations".